commerce_adyen.api.php

```diff
@@ -1,257 +0,0 @@
-<?php
-
-/**
- * @file
- * Commerce Adyen API.
- */
-
-/**
- * Allow alter the data used to create payment authorisation.
- *
- * @param \Commerce\Adyen\Payment\Authorisation\Request $payment
- *   Payment authorisation request that will be sent to Adyen.
- * @param \stdClass $order
- *   Commerce order.
- * @param array $payment_method
- *   Commerce payment method.
- *
- * @see commerce_adyen_redirect_form()
- */
-function hook_commerce_adyen_payment_authorisation_request_alter(\Commerce\Adyen\Payment\Authorisation\Request $payment, \stdClass $order, array $payment_method) {
```

Comment:
Yes I delete all hooks because they were just copies of the D7 version, not a real migration. The module have plugins. One way of extend or alter the functionality is by EventSubscriber, or extend plugins for new implementations.